global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "allowPrerelease": true
```

Comment:
This was needed because the `VSEngSS-MicroBuild2022-1ES` machines are on released VS but we need a bleeding-edge SDK.

Reply:
Should we include `7.0.100-preview.6.22352.1` here?

Reply:
I dithered on this and decided no but evidently I didn't need much persuasion otherwise, just a sec